index.html

```diff
@@ -0,0 +1,297 @@
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200">
+    <link href="https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <header class="header wrapper">
+        <a class="header__logo flex-align-items" href="#">
+            <img class="header__logo-img" src="public/assets/img/logo.svg" alt="logo">
+            Lasles<span class="header__logo-bold-text">VPN</span>
+        </a>
+        <nav>
+            <ul class="header__list">
+                <li class="header__list-element"><a href="#">About</a></li>
+                <li class="header__list-element"><a href="#">Features</a></li>
+                <li class="header__list-element"><a href="#">Pricing</a></li>
+                <li class="header__list-element"><a href="#">Testimonials</a></li>
+                <li class="header__list-element"><a href="#">Help</a></li>
+            </ul>
+        </nav>
+        <div class="header__login-btn">
+            <a class="header__login-btn-sign-in" href="#">Sign In</a>
+            <a class="header__login-btn-sign-up" href="#">Sign Up</a>
+        </div>
+    </header>
+    <main>
+        <section class="main-section">
+            <figure class="main-section__figure wrapper">
+                <img src="./public/assets/img/main-section-first-img.png"
+                    alt="a man with laptop using netflix with vpn">
+                <figcaption class="main-section__figcation">
+                    <h1 class="figcaption__title">Want anything to be easy with <span
+                            class="figcaption__title-highlight">LaslesVPN.</span></h1>
+                    <p class="figcaption__description">Provide a network for all your needs with ease and fun using
+                        Lasles<span class="figcaption__description-highlight">VPN</span> discover interesting
+                        features from us.</p>
+                    <a class="figcaption__btn" href="#">Get Started</a>
+                </figcaption>
+            </figure>
+            <div class="statistics wrapper">
+                <figure class="statistics__details ">
+                    <svg class="details__svg first">
+                        <use xlink:href="./public/assets/img/statistic-section-icons.svg#icon-user"></use>
+                    </svg>
+                    <figcaption>
+                        <h6 class="details__number">90+</h6>
+                        <p class="details__description">Users</p>
+                    </figcaption>
+                </figure>
+                <img src="./public/assets/img/vertical-line.svg" alt="vertical line devides two sections">
```

Comment:
щось тут пішло не так

```suggestion
            <div class="statistics wrapper">
                <figure class="statistics-details ">
                    <svg class="statistics-details__svg first">
                        <use xlink:href="./public/assets/img/statistic-section-icons.svg#icon-user"></use>
                    </svg>
                    <figcaption>
                        <h6 class="statistics-details__number">90+</h6>
                        <p class="statistics-details__description">Users</p>
                    </figcaption>
                </figure>
                <img src="./public/assets/img/vertical-line.svg" alt="vertical line devides two sections">
```